Composer/packages/client/src/types/window.d.ts

```diff
@@ -36,3 +36,11 @@ declare global {
     __csrf__?: string;
   }
 }
+
+export type PageLink = {
```

Comment:
Why is this type included in this file? It is supposed to represent the properties on the window object

Reply:
Oops - I added this to the wrong `types` file. I'll fix this.